spec/swagger_helper.rb

```diff
@@ -81,6 +81,56 @@
               example: 'xccdf_org.ssgproject.content_profile_standard'
             }
           }
+        },
+        rule_result: {
+          type: 'object',
+          required: %w[result],
+          properties: {
+            result: {
+              type: 'string',
+              example: 'passed'
+            }
+          }
+        },
+        rule: {
+          type: 'object',
+          required: %w[title ref_id],
+          properties: {
+            title: {
+              type: 'string',
+              example: 'Record Access Events to Audit Log directory'
+            },
+            ref_id: {
+              type: 'string',
+              example: 'xccdf_org.ssgproject.content_rule_directory_access_'\
+              'var_log_audit'
+            },
+            severity: {
+              type: 'string',
+              example: 'Low'
+            },
+            description: {
```

Comment:
Where is this stuff coming from? This file isn't autogenerated, is it?

Reply:
It's not autogenerated, this is the OpenAPI definition I wrote